CHANGELOG.md

```diff
@@ -27,6 +27,7 @@
 
 - Deserializing SentryEvents with Decodable (#4724)
 - Remove internal unknown dict for Breadcrumbs (#4803) This potentially only impacts hybrid SDKs.
+- Add prettier to pre-commit for markdown and json files (#4846)
```

Comment:
I don't think belongs in the changelog. If anything, write about it in CONTRIBUTING.md or develop-docs/README.md

Reply:
```suggestion
```